### ISSUE TITLE
Documentation for v1 and v2

### ISSUE THREAD

**Issue by mithun** [Sceptre/sceptre#574]
Sceptre's [documentation](https://sceptre.cloudreach.com/latest/) displays the documentation for v2. However, there are still plenty of users on v1 that will benefit from documentation being available on https://sceptre.cloudreach.com/.

Here is what I'm proposing:

1. [Homepage](https://sceptre.cloudreach.com/latest/) still defaults to latest stable version (v2.x)
2. Clearly indicate version number on all pages
3. Allow users to switch between documentation for v1 and v2
4. Create new permanent links for each major version (e.g., https://sceptre.cloudreach.com/v1/, https://sceptre.cloudreach.com/v2/, etc)

**Comment by 1oglop1** [comment]
This is work in progress https://github.com/cloudreach/sceptre/pull/567

**Comment by ngfgrant** [comment]
@mithun totally agree - as @1oglop1 has mentioned this should be fixed with #567 so we will have older docs available in the future.

**Comment by et304383** [comment]
Is this available yet?  Many of us will not be moving to V2 so having access to V1 documentation is crucial.

**Comment by ngfgrant** [comment]
@et304383 You can see from the 2.1.1 Milestone, the Project Board and the PR that this is "In Review" at the moment. Review comments have been made and are being addressed by @1oglop1 when he's got time.

**Comment by et304383** [comment]
When can we expect a link?  :)

**Comment by ngfgrant** [comment]
@et304383 once we get the 2.1.1 branch merged to master - hopefully today.

**Comment by n13k77** [comment]
Maybe it's me but when I currently check the documentation I can only see versions latest, dev, 2.1.4, 2.1.3, 2.1.2 and 2.1.1. So no links to the documentation of v1; am I missing something or is v1 documentation (still) not accessible?

**Comment by ngfgrant** [comment]
Thanks for the reminder @n13k77 the docs move has happened recently. @1oglop1 I think that we can just build the v1 docs and push to the new github docs page manually ?

**Comment by 1oglop1** [comment]
Yes you could build it manually and then push the directory with static files to https://github.com/Sceptre/sceptre.github.io but keep in mind that this would require also fixing `version-helper.js` to enable cross reference from v2.
DM me via Slack for more details.

**Comment by n13k77** [comment]
Any updates?

**Comment by et304383** [comment]
Where is the V1 documentation?  Ugh...

**Comment by ngfgrant** [comment]
Hi guys sorry the publishing of the docs to the new doc site has taken a back seat at the moment but the docs are still available in the v1 branch https://github.com/Sceptre/sceptre/tree/v1/docs/docs

**Comment by 1oglop1** [comment]
I will be able to commit some time to this next week. If all goes well I should be able to put it together.
